pyke/prf_photometry.py

```diff
@@ -0,0 +1,407 @@
+from abc import abstractmethod
+import math
+import scipy
+import numpy as np
+import tqdm
+import sys
+from astropy.io import fits as pyfits
+from oktopus.posterior import PoissonPosterior
+from .utils import channel_to_module_output, plot_image
+
+if sys.version_info[0] == 2:
+    from inspect import getargspec
+    def _get_number_of_arguments(func):
+        return len(getargspec(func).args)
+
```

Comment:
Let's add a comment here to explain what's going on here, and that it can be removed when we remove Python 2 support.